### DISCUSSION TITLE
Your codespace export did not complete. Please try again

### DISCUSSION THREAD

**Discussion by jalerson** [community/community#121730] [Codespaces]
### Select Topic Area

Bug

### Body

Hello,

I'm using GitHub Codespaces with my students, and one of them is having trouble to publish her workspace to a new repository.

<img width="279" alt="Screenshot 2024-04-30 at 19 26 59" src="https://github.com/community/community/assets/583500/68c301e6-8d50-426a-9828-978301ba571b">

As you can see in the screenshot below, she's getting "Your codespace export did not complete. Please try again.".

<img width="1484" alt="Screenshot 2024-04-30 at 19 28 11" src="https://github.com/community/community/assets/583500/59d2003f-48bb-43f9-921a-26bdc800bfcd">

She tried multiple times in different days, and got the same error.

**Comment by 54145a** [comment]
Same issue here.

**Reply by dukesync** [reply]
Did you ever find a solution? 2026 and still getting the same issue.

**Comment by HeitorAugustoLN** [comment]
Same issue.
